guidelines/releases/release-communication.md

```diff
@@ -0,0 +1,52 @@
+# Release communication
+
+This document describes a release process communication.
+
+## Communication channels
+
+The communication Kyma team should be conducted simultaneously on the two important channels: 
+
+- `c4core-xf-team` - main channel to communicate about the release progress. 
+- `c4core-kyma-scrum-masters` - a channel to escalate actions in case when some team lingers with their job. In that case you can also write directly to the proper Scrum Master.
+
+The Kyma community should be notified about the release on the external channel. Our official channel for external announcements in `#general` on the `kyma-community` slack workspace.
+
+## Communication persons
+
+The Kyma team persons which are responsible for the releasing:
+
+- [Jose Cortina](https://github.com/jose-cortina)
+
+## Release master responsibilities
```

Comment:
I'm not sure whether this section should be mentioned in this document. Please check whether it's not already described in the `release-process.md` or the `release-strategy.md`. Maybe these documents are better for info described in this section?